src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -284,7 +285,8 @@ def init_sync_d_interface_tables(db_conn):
     for sai_id, if_name in if_id_map_util.items():
         if (re.match(port_util.SONIC_ETHERNET_RE_PATTERN, if_name) or \
                 re.match(port_util.SONIC_ETHERNET_BP_RE_PATTERN, if_name) or \
-                re.match(port_util.SONIC_ETHERNET_IB_RE_PATTERN, if_name)):
+                re.match(port_util.SONIC_ETHERNET_IB_RE_PATTERN, if_name) or \
+                re.match(port_util.SONIC_ETHERNET_REC_RE_PATTERN, if_name)):
```

Comment:
Can we add unit-test for both the new interface patterns?

Reply:
@SuvarnaMeenakshi sorry, I missed this comment.  Wondering do you know if there is an existing unit test that I can start with?

Reply:
@ysmanman You could add the IB in asic mock db, like the mock data added here:
https://github.com/Azure/sonic-utilities/blob/master/tests/mock_tables/asic0/appl_db.json#L50

Reply:
@SuvarnaMeenakshi Thanks for the pointer!